---
title: "Critical Analysis Tepary Bean"
author: "Sammi Johnson"
date: "2026-01-02"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```
# Cover Page

Author: Sammi Johnson

Matrikelnummer: 8228156

Article Title: The tepary bean genome provides insight into evolution and domestication under heat stress

Article URL/DOI: https://doi.org/10.1038/s41467-021-22858-x

<div style="page-break-after: always">
  <hr>
</div>
# Introduction

* Short information about the paper. Summary of main aims and findings.
  * TBD
* Why did you choose this paper
  * TBD

# Main Body

## Abstract
* Does it reflect content of the paper?
  * Yes, the article reflects the content of the paper, providing a short description of the understood relation between
  tepary and common bean species before elucidating how the genetic research carried out in the study supports the utility
  of this relationship with respect to common bean aridity and heat-resistance breeding efforts in a changing climate.
* Does it describe the main findings?
  * Yes, see above.
* Is it accessible to more general audience?
  * One could argue the abstract is accessible to an audience with a passing knowledge of biology, insofar as
  knowing what a genome is in broad strokes as a corpus of a species' genetic information.

## Introduction
* Does it give sufficient background to understand the rationale for research?
  * Yes, the study presents the rationale upfront as very directly impacting the availability of the world's edible protein, mostly
  from common bean, as dependent on increasing temperature and aridity abiotic stress. Moreover, it makes the case that
  understanding the mechanisms of these resistances (on display in the closely-related tepary bean) will be crucial in maintaining
  and improving common bean crop performance. 
* Does it link/refer to previous research?
  *  Yes, it links to prior research establishing tepary bean as a close enough relative to common bean to serve as a key source of 
  heat and drought tolerance, and that hybridization attempts have succeeded in conferring beneficial traits both to and from common
  bean.

## Methods
* To your knowledge, are the methods appropriate?
  * To my knowledge, yes, their methodology was sound. The germplasm for Frijol Bayo was purified with 3 generations in the screenhouse,
  and W6 selfed for several generations from an existing 1955 USDA accession. Overview:
    * Frijol Bayo:
      * WGS'd (whole-genome shotgun sequenced) with Illumina X10 and HiSeq
      * Standard (long read de novo) sequencing with Pacbio RS platform for contextual scaffolding (lkup)
      * Assembly (nums of results and versions of software all listed):
        * Screen for mito, plast, and phix contam
        * MECAT assembler
        * ARROW
        * JUICER for IDing misjoins, common bean syteny, and hi-c scaffolding
          * misjoins resolved
          * orientation, ordering, and joining of scaffolds
          * signif telomeric region ID'd with TTTAGGG_n repeat
          * remaining scaffolds dealt with
        * BWA-MEM: aligning ~100x Illumina WGS reads
        * GATK UnifiedGenotyper: use aligned WGS reads to correct INDELs and homozyg SNPs in consensus release
        * decontam with blastn and blastx against known microbe proteins

    * W6:
      *  Chromium 10x library
      * DenovoMAGIC2
      * aligned by flanking snp chip markers with BWA scaffolds
      * confirmed order with single oxford nanopore library
  Standard
* Are they described in sufficient details (for example software and software version) to reproduce the research?
  * Yes, all subsections of the 
* Are the methods state of the art or is the approach outdated? (for example short reads used for assembly when long reads could be produced)
  * Most approaches seem up to date, though the platform for long reads (PacBio RS) is outdated by two generations (RSII and Sequel) of SMRT
  sequencing
* Is the data/code publicly available?
  * The data is all available on publicly accessible archives under bioproject PRJNA607288. The versions of all used software were listed, but
  there isn't a repository for the pipeline, so presumably these were processed interactively.
* Can you pinpoint any weaknesses of the study design?
  * The lack of a documented software pipeline, especially with respect to the command line flags used for each step. These aren't provided in
  the paper, and thus must be inferred from the usecases described in the methods section.

## Results
* Can the results section be easily followed? Are there subheadings to help guide the reader?
  * TBD
  * TBD
* Are the figures readable and is the information well presented? Are the legends/captions sufficient to understand the figures even without reading the whole text in detail?
  * TBD
  * TBD
* Do the text and figures complement each other well or are there repetitions which could be
avoided? Are all figures/tables mentioned in the text? Does the text expand on them?
  * TBD
  * TBD
  * TBD
* Are there sections of text or figures/tables which are difficult to understand?
  * TBD
* Do the results appear reliable? Are there sufficient quality checks?
  * TBD
  * TBD
* Can you see any inconsistencies in between the figures and text?
  * TBD
* Are there any potential errors in the results section?
  * TBD

## Discussion
* Does the discussion provide interpretation, described significance of the results?
  * TBD
* Does the interpretation arise logically from the data? Is the discussion interesting? Does it point out any conclusions/inferences not immediately obvious from the results?
  * TBD
  * TBD
  * TBD
* Does the discussion mention limitations of the study and points to future research?
  * TBD

## Conclusion
* Did you like the paper? What did you like about it? What did you not? Could it be mostly understood by more general audience?
  * TBD
  * TBD
  * TBD
* Did you learn anything new?
  * TBD
* Would you recommend it to others? Why yes, why not?
  * TBD