---
title: "Critical Analysis Tepary Bean"
author: "Sammi Johnson"
date: "2026-01-02"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Cover Page

Author: Sammi Johnson

Matrikelnummer: 8228156

Article Title: The tepary bean genome provides insight into evolution and domestication under heat stress

Article URL/DOI: <https://doi.org/10.1038/s41467-021-22858-x>

::: {style="page-break-after: always"}
<hr>

</div>

# Introduction

-   Short information about the paper. Summary of main aims and findings.

-   Why did you choose this paper

    This paper aims to report on the assembly of a reference genome for tepary bean (*P. actufolius*), and provide insight into how its heat and drought resistance works. This research is done explicitly in the context of climate change threatening the most important legume protein for humans, common bean, and how understanding tepary bean in this way may present opportunities for trait development in common bean.

    I grew up in an arid environment and have been long been interested in how crops can manage to survive in those conditions, especially as more of the world faces aridity with climate change.

# Main Body

## Abstract

-   Does it reflect content of the paper?

-   Does it describe the main findings?

-   Is it accessible to more general audience?

    One could argue the abstract is accessible to an audience with a passing knowledge of biology, insofar as knowing what a genome is in broad strokes as a corpus of a species' genetic information.

    The abstract reflects the content of the paper, providing a short description of the findings that establish tepary as a close relative of common bean, and how this could be utilized in heat-resistance breeding efforts for important protein legumes in a changing climate.

## Introduction

-   Does it give sufficient background to understand the rationale for research?

    Does it link/refer to previous research?

    The study presents the rationale upfront as very directly impacting the availability of the world's edible protein, mostly from common bean, as dependent on increasing temperature and aridity abiotic stress. Moreover, it makes the case that understanding the mechanisms of these resistances (on display in the closely-related tepary bean) will be crucial in maintaining and improving common bean crop performance.

    Further, the article links to prior research establishing tepary bean as a close enough relative to common bean to serve as a key source of heat and drought tolerance, and that hybridization attempts have succeeded in conferring beneficial traits both to and from common bean.

## Methods

-   To your knowledge, are the methods appropriate?

-   Are they described in sufficient details (for example software and software version) to reproduce the research?

-   Are the methods state of the art or is the approach outdated? (for example short reads used for assembly when long reads could be produced)

-   Is the data/code publicly available?

-   Can you pinpoint any weaknesses of the study design?

    To the best of my understanding, the authors' methodology was sound. The germplasm for Frijol Bayo was purified with 3 generations in the screenhouse, and W6 selfed for several generations from an existing 1955 USDA accession. Frijol Bayo was sequenced with WGS and PacBio long reads to create the scaffolds for alignment. W6 tepary's genome was built from a Chromium 10x de novo library, aligning reads by flanking illumina SNPs with BWA scaffolds. The two genomes were measured for quality (BUSCO) and size, then annotated against a specific library of plant protein-encoding seqs.

    All subsections of the methods list the assembly, alignment, statistical, and visualization software used, along with their associated versions and source code repositories where available.

    Most approaches seem up to date, though the platform for long reads (PacBio RS) is outdated by two generations (RSII and Sequel) of SMRT sequencing

    The data is all available on publicly accessible archives under bioproject PRJNA607288. The versions of all used software were listed, but there isn't a repository for the pipeline, so presumably these were processed interactively.

    Not all software used has the command line flags listed. Thus, the parameters for those programs must be inferred from the use cases described in the methods section. Additionally the overall pipeline is not in a dedicated repository, leading me to believe that, outside of pre-existing pipelines, the overall process must have been handled interactively.

## Results

-   Can the results section be easily followed? Are there subheadings to help guide the reader?
    -   The results section follows understandably from the abstract, with subheadings marking the crucial findings.
-   Are the figures readable and is the information well presented? Are the legends/captions sufficient to understand the figures even without reading the whole text in detail?
    -   The figures are well presented, given the complexity they encompass, and the legends and captions are succint but sufficient. They used a gene expression network diagram particularly well to explain heat stress response differences between common and tepary bean.
-   Do the text and figures complement each other well or are there repetitions which could be avoided? Are all figures/tables mentioned in the text? Does the text expand on them?
    -   Likewise, the underlying text and said figures complement each other well, certainly helping me understand some less intuitive relationships. All figures are referenced when the results reaches a point where text begins to hit its limit as an expressive medium. And, for those who are familiar with the analytical techniques and measurements presented in the figures, the text provides needed context.
-   Are there sections of text or figures/tables which are difficult to understand?
    -   Most of the results text and visuals are understandable intuitively. The paper does a good job making it clear which bean is being discussed at any given time, so it avoids that possible source of confusion. In figure 1d, the estimated divergence time chart between the bean species, I had a difficult time seeing the overlapping thin lines clearly to find any particular bean-bean relationship. That being said, I understood the point it was making with respect to the text (that tepary is a recently divergent sister taxon of common bean) and I don't think that could have been avoided without compromising clarity further.
-   Do the results appear reliable? Are there sufficient quality checks?
    -   The results also appear reliable, given the quality checks undertaken with the genetic samples, of the genome assemblies themselves, and the authors' awareness that differences
-   Can you see any inconsistencies in between the figures and text?
    -   No, as far as I can tell, the figures and text match closely with each other
-   Are there any potential errors in the results section?
    -   There weren't errors in the results as far as I could ascertain.

## Discussion

-   Does the discussion provide interpretation, described significance of the results?
    -   In the discussion, the article describes the significance
-   Does the interpretation arise logically from the data? Is the discussion interesting? Does it point out any conclusions/inferences not immediately obvious from the results?
    -   TBD
    -   TBD
    -   TBD
-   Does the discussion mention limitations of the study and points to future research?
    -   TBD

## Conclusion

-   Did you like the paper? What did you like about it? What did you not? Could it be mostly understood by more general audience?
    -   TBD
    -   TBD
    -   TBD
-   Did you learn anything new?
    -   Yes, I did! I learned how the regulatory cascade works in beans of this clade to confer heat stress resistance. I also learned to what extent these species are related, letting hybridization occur (with embryo rescue, but nonetheless).
-   Would you recommend it to others? Why yes, why not?
    -   In spite of minor shortcomings in reproducibility, and perhaps some older sequencing technologies, the fundamental principles of study seem solid to me, and I'd recommend anyone interested in abiotic stress trait evolution read it.

:::
:::
